### DISCUSSION TITLE
How to quickly merge configs?

### DISCUSSION THREAD

**Discussion by sergiomarotco** [zricethezav/gitleaks#1126] [Q&A]
How do you merge your local config file with the [official config](https://github.com/gitleaks/gitleaks/blob/master/config/gitleaks.toml) file that is updated in this repository?
What is the best way for me to clone a [official config](https://github.com/gitleaks/gitleaks/blob/master/config/gitleaks.toml) from a this repository and add all the exceptions I created to it in my local config?

**Comment by sergiomarotco** [comment]
Maybe it would be nice to modify Gitleaks so that all exceptions are added at the end of the config. Then is it enough just to clone the official config and transfer exceptions from the old local config to the end?

**Reply by weineran** [reply]
@sergiomarotco check out this blog post for some ideas, especially the section titled "Extending a Central Config"
https://blog.gitleaks.io/stop-leaking-secrets-configuration-2-3-aeed293b1fbf
